WinUIGallery/Controls/ControlExample.xaml

```diff
@@ -4,9 +4,10 @@
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:contract7Present="using:Microsoft.UI.Xaml.Controls?IsApiContractPresent(Windows.Foundation.UniversalApiContract,7)"
     xmlns:controls="using:WinUIGallery.Controls"
+    xmlns:controls1="using:WinUIGallery.DesktopWap.Controls"
```

Comment:
Besides code reordering, did anything change in this file?

Reply:
Nope